.circleci/config.yml

```diff
@@ -1,60 +1,43 @@
 version: 2
 
+dockerbuild: &dockerbuild
```

Comment:
what does the `&dockerbuild` token mean

Reply:
it makes it an anchor to be referenced later